www-offline/layouts/search/section.html

```diff
@@ -0,0 +1,165 @@
+{{ define "main" }}
+{{ $pathToRoot := strings.TrimSuffix "/" (partial "path_to_root.html" .Permalink) }}
+{{ $websites := where site.Pages ".Params.content_type" "==" "website" }}
+{{ block "header" . }}
+{{ partial "header" . }}
+{{ end }}
+<div class="search-wrapper">
+  <div class="search-page row justify-content-center">
+    <div class="search-results-area col-12 py-4">
+      <div class="container-fluid">
+        <main>
+          <div class="container-fluid p-0">
+            <div class="row m-0">
+              <div id="search-page" class="w-100">
+                <div class="container">
+                  <div class="search-box py-sm-5 py-md-7 py-lg-5 row">
+                    <div class="col-lg-3"></div>
+                    <div class="col-lg-6 search-box-inner d-flex flex-column align-items-center mb-2 mb-sm-5 mb-md-4">
+                      <h1>Explore OpenCourseWare</h1>
+                      <div class="w-100 d-flex flex-column align-items-center search-input-wrapper mt-5">
+                        <div class="w-100">
+                          <form role="search" class="search-box d-flex flex-direction-row">
+                            <input
+                              id="search-input"
+                              class="w-100 pl-2"
+                              type="text"
+                              aria-label="Search"
+                              placeholder="Search OpenCourseWare"
+                              />
+                            <button id="search-button" type="submit" class="py-2 px-3">Search</button>
```

Comment:
```suggestion
                            <button id="search-button" type="button" class="py-2 px-3">Search</button>
```

Then you won't need to prevent default down below.